qa/L0_multiple_ports/multi_port_tests.py

```diff
@@ -0,0 +1,115 @@
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+# OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+import argparse
+import numpy as np
+import os
+from builtins import range
+from tensorrtserver.api import *
+import tensorrtserver.api.server_status_pb2 as server_status
+
+FLAGS = None
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-v', '--verbose', action="store_true", required=False, default=False,
+                        help='Enable verbose output')
+    parser.add_argument('-u', '--url', type=str, required=False, default='localhost:',
+                        help='Inference server URL. Default is localhost:.')
+    parser.add_argument('-i', '--protocol', type=str, required=False, default="http",
+                        help='Protocol ("http"/"grpc") used to ' +
+                        'communicate with inference service. Default is "http".')
+    parser.add_argument('-p', '--port', type=int, required=False, default=-1,
+                        help="The port for the server to listen on for all HTTP requests.")
+    parser.add_argument('-sp', "--status_port", type=int, required=False, default=-1,
+                        help="The port for the server to listen on for HTTP Status requests.")
+    parser.add_argument('-hp', "--health_port", type=int, required=False, default=-1,
+                        help="The port for the server to listen on for HTTP Health requests.")
+    parser.add_argument('-pp', "--profile_port", type=int, required=False, default=-1,
+                        help="The port for the server to listen on for HTTP Profile requests.")
+    parser.add_argument('-ip', "--infer_port", type=int, required=False, default=-1,
+                        help="The port for the server to listen on for HTTP Infer requests.")
+
+    FLAGS = parser.parse_args()
+    FLAGS.status_port = FLAGS.port if FLAGS.status_port==-1 else FLAGS.status_port
+    FLAGS.health_port = FLAGS.port if FLAGS.health_port==-1 else FLAGS.health_port
+    FLAGS.profile_port = FLAGS.port if FLAGS.profile_port==-1 else FLAGS.profile_port
+    FLAGS.infer_port = FLAGS.port if FLAGS.infer_port==-1 else FLAGS.infer_port
+
+    protocol = ProtocolType.from_str(FLAGS.protocol)
+
+    # We use a simple model that takes 2 input tensors of 16 integers
+    # each and returns 2 output tensors of 16 integers each. One
+    # output tensor is the element-wise sum of the inputs and one
+    # output is the element-wise difference.
+    model_name = "simple"
+    model_version = -1
+    batch_size = 1
+
+    # Create the inference context for the model.
```

Comment:
This comment seems like from an outdated cur and paste since you are creating many contexts here not just infer